src/install/install.zig

```diff
@@ -11190,13 +11189,10 @@ pub const PackageManager = struct {
             try workspace_package_json_file.pwriteAll(source, 0);
             std.posix.ftruncate(workspace_package_json_file.handle, source.len) catch {};
             workspace_package_json_file.close();
+        }
 
-            if (subcommand == .remove) {
-                if (!any_changes) {
-                    Global.exit(0);
-                    return;
-                }
-
+        if (manager.options.do.install_packages) {
```

Comment:
The previous minimal `bun remove` logic triggers on `do.write_package_json`, but bun unlink defaults to `--no-save` and I think running removal/unlinking logic on `do.install_packages` makes more sense.